code/cgame/cg_localents.c

```diff
@@ -3313,3 +3313,16 @@ void CG_ClearFxExternalForces (void)
 	}
 }
 #endif
+
+
+void CG_HideDamagePlums (int client, int weapon) {
```

Comment:
Thanks.  Is CG_HideDamagePlums() just used to hide self damage?  Why not just avoid adding them at all?

Reply:
>  Is CG_HideDamagePlums() just used to hide self damage?

No. It used for different case. For example we have 3 plasma hits. Without using CG_HideDamagePlums() and enabled damage sum hack it will show 3 damage plums. 20, 40 and 60. When all these damage plums are shown, it is hardly readable how many damage dealt.

When usin CG_HideDamagePlums, on second hit it will show damage plum with "40" and hide previous damage plum "20". On third hit it will show damage plum "60" and hide previous damage plum "40".